app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -43,6 +45,10 @@ class Scene extends React.Component {
       const loadedWorld = this.res.resurrect(world);
       Engine.merge(this.engine, { world: loadedWorld });
     }
+    const allBodies = Composite.allBodies(this.engine.world);
```

Comment:
probably don't need to keep track of allBodies here, can just filter/map from here